### PR TITLE
some fixes in the doc of grangercausalitytests

### PR DESCRIPTION
**Pull request by helderc**
I just did some fixes in the documentation.

### CODE REVIEW

**Comment by josef-pkt** [comment]
Thanks

looks good to me

File: statsmodels/tsa/stattools.py [hunk 1]
```diff
@@ -1330,10 +1330,10 @@ def grangercausalitytests(x, maxlag, addconst=True, verbose=True):
     >>> import numpy as np
     >>> data = sm.datasets.macrodata.load_pandas()
     >>> data = data.data[["realgdp", "realcons"]].pct_change().dropna()
-
+    >>>
```
Comment (bashtage) [inline statsmodels/tsa/stattools.py:1333]: This isn't needed.

File: statsmodels/tsa/stattools.py [hunk 2]
```diff
@@ -1330,10 +1330,10 @@ def grangercausalitytests(x, maxlag, addconst=True, verbose=True):
...
     # All lags up to 4
```
Comment (bashtage) [inline statsmodels/tsa/stattools.py:1334]: Remove the leading # here and align All with the indentation.

File: statsmodels/tsa/stattools.py [hunk 3]
```diff
@@ -1330,10 +1330,10 @@ def grangercausalitytests(x, maxlag, addconst=True, verbose=True):
...
     >>> gc_res = grangercausalitytests(data, 4)
-
+    >>>
```
Comment (bashtage) [inline statsmodels/tsa/stattools.py:1336]: Same here.  Gaps are find, but there shouldn't be a commend below. So please remove the #.

**Review by bashtage — CHANGES_REQUESTED** [review]
Thanks for the PR. Some small cleanups would help.

File: statsmodels/tsa/stattools.py (hunk 1 above)
Comment (helderc) [reply statsmodels/tsa/stattools.py:1333]: I'm not that familiar with Git, is that possible to change the PR?

Comment (helderc) [reply statsmodels/tsa/stattools.py]: never mind. I guess I fixed it as requested.

**Comment by bashtage** [comment]
Many thanks.

**Pull request merged**